creator/ingest_runs/signals.py

```diff
@@ -13,3 +16,31 @@ def ingest_run_pre_save(sender, instance, *args, **kwargs):
     if instance.versions.count() > 0:
         instance.name = instance.compute_name()
         instance.input_hash = instance.compute_input_hash()
+
+@receiver(post_transition, sender=IngestRun)
+def ingest_run_post_transition(sender, instance, name, source, target,
+    *args, **kwargs):
+    """
+    Create a corresponding Event after the status of an IngestRun is changed.
+    """
+    TARGET_EVENT_TYPES = {
+        "running": "IR_STA",
+        "complete": "IR_COM",
+        "canceled": "IR_CAN",
+        "failed": "IR_FAI",
+    }
+    username = getattr(instance.creator, "display_name", "Anonymous user")
+    message = (
```

Comment:
Couple things about the message: 
1. Could you add the status into the message somewhere so the event timeline tells us what happened to the ingest ru?
2. It probably only makes sense to display the username in the message if the user started the ingest run or canceled it. It sounds weird to say something like "Gio changed the status of IngestRun to FAILED" haha